utils/scripts/ocm/README.md

```diff
@@ -0,0 +1,100 @@
+# Provision Openshift Dedicated Cluster in AWS
```

Comment:
The script also does other things more than provisioning OSD cluster. So, Can we say like , Openshift Dedicated Cluster Operations in AWS ?